addons/pos_event/models/event_event.py

```diff
@@ -0,0 +1,82 @@
+from odoo import api, fields, models
+from itertools import groupby
+from operator import itemgetter
+from functools import reduce
+from collections import defaultdict
+
+
+class Event(models.Model):
+    _name = 'event.event'
+    _inherit = ['event.event', 'pos.load.mixin']
+
+    pos_order_lines_ids = fields.One2many('pos.order.line', 'event_id', string="All pos order lines pointing to this event")
+    pos_price_subtotal = fields.Monetary("PoS sales (Tax Excluded)", compute='_compute_pos_price_subtotal')
+    image = fields.Image("Image", max_width=1024, max_height=1024)
+
+    @api.model
+    def _load_pos_data_domain(self, data):
+        return [('is_finished', '=', False), ('is_ongoing', '=', False), ('event_ticket_ids', '!=', False)]
```

Comment:
Is it expected that you cannot buy tickets for ongoing events ?

Reply:
Shouldn't the domain be based more on `start_sale_datetime` and `end_sale_datetime` fields from `event.ticket` ?